bin/metrics-aggregate.rb

```diff
@@ -76,12 +81,12 @@ def api_request(resource)
     end
     req = Net::HTTP::Get.new(resource)
     r = http.request(req)
-    JSON.parse(r.body)
+    ::JSON.parse(r.body)
```

Comment:
This is needed to prevent conflicting class name resolution with name JSON. This will ensure that the json package will be used rather than the imported `Sensu::Plugin::Metric::CLI::JSON` class.

Reply:
yup I have had to do similar things in chef as well.